README.md

```diff
@@ -24,6 +24,8 @@ You can build the `grace` executable using `cabal`:
 $ cabal build exe:grace
 ```
 
+Note: For newer versions of cabal, use `cabal new-build exe:grace`
+
```

Comment:
I think it's actually _older_ versions (< 3) where Nix-style builds aren't used by default. In cabal v3, `build`, `new-build` and `v2-build` are the same, AFAIK.

Also, AFAIK, the recommendation is to use `v2-build` since `new-build` may eventually be used to provide different semantics.

Reply:
Yeah, my understanding is that the default behavior for newer versions of `cabal` is `cabal v2-build` a.k.a. `cabal new-build`

Also, I think "old build" still works, too.  If it doesn't, though, let me know and I can try to fix it

Reply:
I didn't create an issue for this -- I brought it up casually in #35, but `cabal build exe:grace` returns this:

```
$ cabal build exe:grace
Warning: The build command is a part of the legacy v1 style of cabal usage.

Please switch to using either the new project style and the new-build command
or the legacy v1-build alias as new-style projects will become the default in
the next version of cabal-install. Please file a bug if you cannot replicate a
working v1- use case with the new-style commands.

For more information, see: https://wiki.haskell.org/Cabal/NewBuild

Resolving dependencies...
Warning: solver failed to find a solution:
Could not resolve dependencies:
[__0] trying: grace-1.0.0 (user goal)
[__1] unknown package: unordered-containers (dependency of grace)
[__1] fail (backjumping, conflict set: grace, unordered-containers)
After searching the rest of the dependency tree exhaustively, these were the
goals I've had most trouble fulfilling: grace, unordered-containers
Trying configure anyway.
Configuring grace-1.0.0...
cabal: Encountered missing dependencies:
Earley -any,
ansi-terminal -any,
generic-lens -any,
http-client -any,
http-client-tls -any,
http-types -any,
insert-ordered-containers -any,
lens -any,
megaparsec -any,
modern-uri -any,
optparse-applicative -any,
parser-combinators -any,
prettyprinter -any,
prettyprinter-ansi-terminal -any,
repline -any,
safe-exceptions -any,
scientific -any,
string-interpolate -any,
terminal-size -any,
text >=1.2.4.0,
unordered-containers -any```

Reply:
as @sjakobi said, my cabal version is 2.4.0.0 -- updating per this suggestion

```suggestion
Note: For older versions of cabal (e.g. version <3), use `cabal new-build exe:grace`

```